pkg/cluster/internal/kubeadm/config.go

```diff
@@ -356,6 +372,10 @@ controlPlaneEndpoint: "{{ .ControlPlaneEndpoint }}"
 # to the cluster after rewriting the kubeconfig to point to localhost
 apiServer:
   certSANs: [localhost, "{{.APIServerAddress}}"]
+  {{ if .PodSecurityPolicy -}}
+  extraArgs:
+    enable-admission-plugins:  PodSecurityPolicy,LimitRanger,ResourceQuota,AlwaysPullImages,DefaultStorageClass
```

Comment:
Uh, you think AlwaysPullImages might be why it wasn't using the preloaded images? 🙃

Reply:
🤦‍♂